### PR TITLE
feat (Automated log work input (status and works completion) and added userid

### PR DESCRIPTION
**Pull request by murlidharpathakprogrammer**
### Functionality:
Automated log work including prefilled status, work completion and added userid

### Risk level:
- [ ] high 
- [ ] medium
- [X] low

### How to test:
ng serve
firebase emulators:start

### CODE REVIEW

File: src/app/body/task-details/log-work/log-work.component.ts [hunk 1]
```diff
@@ -17,11 +17,13 @@ export class LogWorkComponent implements OnInit {
   @ViewChild('form') form: NgForm;
   @Input('task') task: Tasks
   @Output() logWorkCompleted = new EventEmitter<{ completed: boolean }>();
-  
+
   componentName: string = "LOG-WORK";
+
```
Comment (k0malSharma) [inline src/app/body/task-details/log-work/log-work.component.ts:22]: Remove empty line

**Review by k0malSharma — CHANGES_REQUESTED** [review]
Make the required changes

**Review by murlidharpathakprogrammer — COMMENTED** [review]
updated the required changes and removed the unnecessary empty lines

**Review by k0malSharma — APPROVED** [review]
Ship it

**Comment by k0malSharma** [comment]
Resolve Merge Conflict ASAP

**Pull request merged**